### ISSUE TITLE
Grass: build fails (missing dependency FFTW)

### ISSUE THREAD

**Issue by mucaho** [NixOS/nixpkgs#9160]
This is a follow-up of #9121 .
I installed `SAGA` and I was able to call these additional geo-algorithms from `QGis`.

However, `Grass` doesn't work unfortunately. @vbgl added a `broken` label to the [pkg description](https://github.com/NixOS/nixpkgs/blob/158e1cfdd0ddb8380e266dcf16a4167455335bc3/pkgs/applications/misc/grass/default.nix) in #4898. I tried to build it, the first error that pops up is about the `Fastest Fourier Transform in the West` library.
```
checking whether to use FFTW... yes
checking for location of FFTW includes... /nix/store/sj1ldf5cnmyp5csbk3vld4yhqdpy48l8-fftw-single-3.3.4/include
checking for fftw3.h... yes
checking for location of FFTW library... /nix/store/sj1ldf5cnmyp5csbk3vld4yhqdpy48l8-fftw-single-3.3.4/lib
checking for fftw_execute in -lfftw3... no
checking for fftwnd_one in -lfftw... no
checking for fftwnd_one in -ldfftw... no
configure: error: *** Unable to locate FFTW library.
```
There is a `fftwSupport` config option, however it is not used anywhere in the nix expression.
Maybe it could be disabled by default and depending on this config option the respective configure option can be set?
* `./configure --with-fftw`
* `./configure --without-fftw`

-----------------------------------------

Link **[Compile and Install Grass](http://grasswiki.osgeo.org/wiki/Compile_and_Install)**
Link **[Requirements for compiling Grass](http://grass.osgeo.org/grass64/source/REQUIREMENTS.html)**
Link **[Grass INSTALL file](http://grass.osgeo.org/grass64/source/INSTALL)**

--------------------

Link **[How to enable Grass functionality inside QGIS](http://docs.qgis.org/2.0/ca/docs/user_manual/processing/3rdParty.html#grass)** says:
> If you are running Linux, you just have to make sure that GRASS is correctly installed, and that it can be run without problem from a console.

Interesting link for future consideration - integrating grass data in QGis **[Compile and install GDAL-GRASS plugin](http://grasswiki.osgeo.org/wiki/Compile_and_install_GRASS_and_QGIS_with_GDAL/OGR_Plugin)**

**Comment by pSub** [comment]
What part of this issue is still relevant? (If any)

**Comment by mucaho** [comment]
@pSub 
Grass successfully compiles and runs - as fixed by #9931.

Still wasn't able to use Grass geo-algorithms from QGIS even after that PR was merged.
Seems like this is a very specific use-case, if there will be any need in future for someone else, this could be investigated further?
